donate/templates/403.html

```diff
@@ -12,10 +12,12 @@ <h1 class="error__heading heading heading--primary">{% trans "Action Required -
     </div>
     <div class="layout__container">
         <div class="layout__half-col error__container">
-            {% blocktrans %}
-              <p class="error__message">Please disable your VPN and visit <a href="https://www.donate.mozilla.org"> donate.mozilla.org </a> again.</p>
-            {% endblocktrans %}
-          <p class="error__message">
+            <p class="error__message">
+                {% blocktrans with url="https://www.donate.mozilla.org" trimmed %}
+                Please disable your VPN and visit <a href="{{ url }}">donate.mozilla.org</a> again.
+                {% endblocktrans %}
```

Comment:
Just noticed that the URL has `www` in front of it, which won't resolve, so let's slip a fix for that into this PR too... but as a question, if the URL itself a var, would it make sense to make the URL's text equivalent a variable too, so folks can't change that accidentally?
```suggestion
                {% blocktrans with url="https://donate.mozilla.org" site="donate.mozilla.org" trimmed %}
                Please disable your VPN and visit <a href="{{ url }}">{{ site}}</a> again.
                {% endblocktrans %}
```

Reply:
Yup that makes sense, it’s even better :) 
And your second suggestion seem to work pretty well, we can probably go with that!
I’ll update the PR, thanks!